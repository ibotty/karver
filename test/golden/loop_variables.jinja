{% for item in listVar %}
 * item #{{loop.index}} (- {{loop.revindex}}) / {{loop.length}}, 0-based: {{loop.index0}} (- {{loop.revindex0}})
    {% if loop.last %}is the last{% endif %}{%if loop.first %}is the first{% endif %}

{% endfor %}
