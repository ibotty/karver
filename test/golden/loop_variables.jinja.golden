 * item #1 (- 2) / 2, 0-based: 0 (- 1)
    is the first
 * item #2 (- 1) / 2, 0-based: 1 (- 0)
    is the last
